main.py

```diff
@@ -188,53 +215,35 @@ def get_table(details):
 
     return tb
 
-
 def populate_data(specific_dep=None):
-    if not os.getenv("JSESSIONID"):
-        print("ERROR: Please set environment variable JSESSIONID!")
-        sys.exit(1)
-
-    cookie = {"JSESSIONID": os.getenv("JSESSIONID")}
+    headers = {
```

Comment:
These headers seem to be repeating. Perhaps make it a constant somewhere and import?

Reply:
Okay, I'll make the changes